AppKit/CPControl.j

```diff
@@ -1056,6 +1056,20 @@ var CPControlBlackColor = [CPColor blackColor];
 
 @end
 
+@implementation CPControl (CPTrackingArea)
+
+- (void)updateTrackingAreas
+{
+    [self removeAllTrackingAreas];
+    
+    [self addTrackingArea:[[CPTrackingArea alloc] initWithRect:CGRectMakeZero()
```

Comment:
The interesting thing is that the Cocoa docs say that when CPTrackingInVisibleRect is used, AppKit doesn't need to call updateTrackingAreas, but in actual fact it does.